### ISSUE TITLE
Run as daemon

### ISSUE THREAD

**Issue by fernandomacho** [ggml-org/whisper.cpp#802]
Hi, I think a nice feature would be to have the ability to run whisper.cpp as a daemon, so that model load times could be avoided. I don't know how complicated it would be to do this and expose a REST API to be able to send through a POST in an audio file

**Comment by cmp-nct** [comment]
By loading the model through mmap you basically have that behavior.
Loading times are reduced to the minimum as long as no other process takes the memory away.

**Comment by fernandomacho** [comment]
True, but right now this is not implemented in the main project right? It's only in the talk-llama example, is it correct? Is it planned to include it in the main project?

**Comment by emcodem** [comment]
If you are not able to make your own program and use whispercpp from API, you could still do it using the existing example binary stream.exe. You'd "just" need to manage to route what you want to translate to a virtual microphone.
